content/press/press-content.md

```diff
@@ -1,11 +1,34 @@
 <!--This is the main content file to edit for this page. It is embedded in src/pages/press.js-->
 
-**Press:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
 
-## Images from events
+## Resources for press/media personnel
+**For inquiries and information:** [Contact](info@environmentalenforcementwatch.org) a member of EDGI's EEW project team.
+
+**Images from events**
 ![EEW public workshop participants](./eew-public-workshop.png)
 
-## Logos
+**Logos**
 ![EEW logo](../images/logos/eew-logo.png)
 ![EEW web icon](./eew-web-icon.png)
 ![EDGI logo](./EDGI-logo.png)
+
+## EEW in the News - Recent Press Coverage of Our Work
+**EEW project covered: [More Permission to Pollute: The Decline of EPA Enforcement and Industry Compliance during COVID](https://envirodatagov.org/more-permission-to-pollute-the-decline-of-epa-enforcement-and-industry-compliance-during-covid/)**
```

Comment:
I'm not sure what "EEW project covered" means, is there another way to phrase?

Reply:
I was unsure about that phrasing as well. Just trying to delineate that the coverage listed there is specific to the covid report, as opposed to the congressional reports project or the workshops generally.

Reply:
I see. This could be fine, or maybe it's better to do a subheading "Reporting on the EEW COVID Report" and link to the COVID report?